index.html

```diff
@@ -3602,13 +3634,24 @@ <h3>Parsing content</h3>
           </li>
           <li>
             Let |record:NDEFRecord| be the result of <a>parse an NDEF record</a>
-            on |ndef|.
+            given |ndef| and |context|.
           </li>
           <li>
             If |record| is not `null`, <a>append</a> |record| to |records|.
           </li>
           <li>
-            If |messageEnd| is `true`, abort these sub-steps.
+            If |messageEnd| is `true`,
+              <ol>
+                <li>
+                  If |context| is not `"smart-poster"`, abort these sub-steps.
+                </li>
+                <li>
+                  If |records| does not contain a <a>URI record</a>
```

Comment:
Ditto. More than one URI record should NOT be OK.

Reply:
fixed